src/assets/i18n/en.json

```diff
@@ -807,8 +807,8 @@
   "ID": "id",
   "USER_CREATION_LIMIT_EXCEEDED": "User Creation limit exceeded",
   "DETAIL_TUTORIAL": "{{%appName}}'s library has a host of {{%explainVideos}}, {{%quesBanks}} and other {{%interactiveMaterial}}. Click on a textbook to start exploring the library!",
-  "CHARACTERS_REMAINING": "Characters remaining",
-  "I_AGREE_TO": "I agree to {{app_name}}",
+  "CHARACTERS_REMAINING": "{{character_count}} Characters remaining",
+  "I_AGREE_TO": "I agree to {{app_name}} ",
```

Comment:
Add the space after name in html not here.

Reply:
The space is added in this PR only.  check line 811